src/picongpu/include/algorithms/KinEnergy.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2017 Heiko Burau
```

Comment:
feel free to keep my name here since I (and others, see your source) designed the algorithm with its threshold nastiness